packages/react/@types/react/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line no-restricted-imports
+export * from 'react'
+
+declare module 'react' {
+  interface CSSProperties {
```

Comment:
can you elaborate as to why this is needed? 🙏

Reply:
Oh, forgot to mention it in the description, this updates the definition of CSSProperties to allow for CSS variables, so that we can just used `React.CSS properties` as the type in https://github.com/primer/react/pull/6169/files#diff-c755511099c2739a4276dc3b714e3ad5083385129f4fb3a461115179c50cbcbcR52, instead of `{[key: string]: string}` which is a bit too generic.

Reply:
Actually looks like its causing the build to fail, so reverting that part of the change